config/development.js

```diff
@@ -6,5 +6,6 @@ module.exports = {
   DB_HOST: 'localhost',
   DB_PORT: 5432,
   DB_PASSWORD: '',
-  DB_USER: 'sfmovies_user'
+  DB_USER: 'sfmovies_user',
+  DB_DEBUG: true
```

Comment:
minor thing that I didn't do correctly in miyagi, but we try to alphabetize the keys in the config objects because when they become much larger, like in api, it's easier to find a key when it's alphabetized.